### PR TITLE
update the udn testing

### PR DESCRIPTION
**Pull request by JacobTanenbaum**
we updated the udn testing to test networks defined by NAD and UDN and needed to update the test names accordingly

this is needed for https://github.com/openshift/origin/pull/29041

### CODE REVIEW

File: pkg/components/networking/ovnkubernetes/component.go [hunk 1]
```diff
@@ -38,6 +38,16 @@ var OvnKubernetesComponent = Component{
...
+			" [sig-network][OCPFeatureGate:NetworkSegmentation][Feature:UserDefinedPrimaryNetworks] EndpointSlices mirroring when using openshift ovn-kubernetes mirrors EndpointSlices managed by the default controller for namespaces with user defined primary networks L3 dualstack primary UDN, host-networked pods [Suite:openshift/conformance/parallel]":   "[sig-network][OCPFeatureGate:NetworkSegmentation][Feature:UserDefinedPrimaryNetworks] EndpointSlices mirroring when using openshift ovn-kubernetes created using NetworkAttachmentDefinitions mirrors EndpointSlices managed by the default controller for namespaces with user defined primary networks L3 primary UDN, host-networked pods [Suite:openshift/conformance/parallel]",
```
Comment (kyrtapz) [inline pkg/components/networking/ovnkubernetes/component.go:46]: is this extra space in the begging intentional?

File: pkg/components/networking/ovnkubernetes/component.go [hunk 2]
```diff
@@ -38,6 +38,16 @@ var OvnKubernetesComponent = Component{
...
+			"[sig-network][OCPFeatureGate:NetworkSegmentation][Feature:UserDefinedPrimaryNetworks] when using openshift ovn-kubernetes can perform east/west traffic between nodes for two pods connected over a L3 dualstack primary UDN [Suite:openshift/conformance/parallel]":                                                                                   "[sig-network][OCPFeatureGate:NetworkSegmentation][Feature:UserDefinedPrimaryNetworks] when using openshift ovn-kubernetes created using NetworkAttachmentDefinitions can perform east/west traffic between nodes two pods connected over a L3 primary UDN [Suite:openshift/conformance/parallel]",
```
Comment (kyrtapz) [inline pkg/components/networking/ovnkubernetes/component.go:48]: Is this correct? I don't see `[sig-network][OCPFeatureGate:NetworkSegmentation][Feature:UserDefinedPrimaryNetworks] when using openshift ovn-kubernetes can perform east/west traffic between nodes for two pods connected over a L3 dualstack primary UDN` in https://github.com/openshift/origin/pull/29041/files#diff-38508ff519d14ea6d62a809a11f00dbbe6c1567e43cea15bc4cd0f699c32edd6

File: pkg/components/networking/ovnkubernetes/component.go (hunk 1 above)
Comment (JacobTanenbaum) [reply pkg/components/networking/ovnkubernetes/component.go]: the space was not intentional, good spot removed it.

File: pkg/components/networking/ovnkubernetes/component.go (hunk 2 above)
Comment (JacobTanenbaum) [reply pkg/components/networking/ovnkubernetes/component.go]: you are right the wording is not the same... I corrected it "...east/west traffic between nodes two pods..."

**Comment by kyrtapz** [comment]
/lgtm
This has to land after https://github.com/openshift/origin/pull/29041 right?

**Pull request merged**